### PR TITLE
[Add] CRM architect

### PR DESCRIPTION
**Pull request by mathi123**

### CODE REVIEW

**Comment by max3903** [comment]
.travis.yml, oca_dependencies.txt and base_construction_architect are already in another PR.

**Comment by mathi123** [comment]
@max3903 : #10 is a branch of #8 ...

File: crm_construction_architect/README.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+===========
+
+This module allows you to assign an architect to a lead or opportunity.
+
+Installation
+============
+
+To install this module, you need to:
+
+* have construction_architect_base available
```
Comment (gurneyalex) [inline crm_construction_architect/README.rst]: wrong module name, and I'd remove the paragraph altogether as this is handled by odoo automatically.

File: crm_construction_architect/oca_dependencies.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# List the OCA project dependencies, one per line
+
+construction_architect_base https://github.com/OCA/vertical-construction
```
Comment (gurneyalex) [inline crm_construction_architect/oca_dependencies.txt]: remove this line: vertical-construction is where we are so no need to list it in the project dependencies.

File: crm_construction_architect/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+##############################################################################
+{
+    'name': "OCA Construction: Architect CRM",
+    'summary': """Indicate the architect of a lead/opportunity.""",
+    'description': """
+        This module allows you to:
+            - Identify the architect of a lead/opportunity.
+            - Filter/group/search opportunities by architect.
+    """,
+    'author': "Mathias Colpaert",
```
Comment (gurneyalex) [inline crm_construction_architect/__openerp__.py]: please add "Odoo Community Association (OCA)" to the list of authors.

**Comment by gurneyalex** [comment]
:+1:

**Comment by max3903** [comment]
:+1:

**Comment by mathi123** [comment]
Thx! :+1:

**Pull request merged**